.github/workflows/evergreen.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
     with:
       platform: evergreen-x64
       nightly: ${{ github.event.inputs.nightly }}
-      run_api_leak_detector: true
+      run_api_leak_detector: false
```

Comment:
It seems to me like we should have some way to still run the api leak detector or we should remove it entirely?

Reply:
yeah, I filed feature request b/307809538